index.html

```diff
@@ -0,0 +1,261 @@
+<html>
+<head>
+  <style>
+  table {
+      border-collapse: collapse;
+      table-layout: fixed;
+  }
+  
+  table, th, tr, td{
+      border: 1px solid black;
+      height: 30px;
+  }
+  
+  table, th{
+      width: 100%;
+      font-size: 25px;
+  }
+  
+  th, a{
+     cursor: default; 
```

Comment:
There is no indication whatsoever that clicking on the title will sort the table. No triangle, no cursor change. If I weren't the one who gave you the exercise, I wouldn't have known the table was sortable.